src/deprecatedCompat/4.8/mergeDecoratorsAndModifiers.ts

```diff
@@ -5,10 +5,28 @@
 //     - error: 5.0
 namespace ts {
     export interface Node {
-        /** @deprecated `decorators` has been merged with `modifiers` on the declarations that support decorators. */
-        readonly decorators?: NodeArray<Decorator> | undefined;
+        /**
+         * @deprecated `decorators` has been merged with `modifiers` on the declarations that support decorators.
+         * To test whether a `Node` can legally have decorators, use the `ts.canHaveDecorators()` function. To read the
+         * decorators of a `Node` that can legally have decorators, use the `ts.getDecorators()` function.

```

Comment:
I got very confused by the last sentence. I'd recommend rewriting it:

```suggestion
         * To test whether a `Node` can legally have decorators, use the `ts.canHaveDecorators()` function.
         * Use `ts.getDecorators()` to get decorators from `Node`s that have them.
```